base/applications/network/tracert/tracert.cpp

```diff
@@ -317,8 +321,19 @@ DecodeResponse(
         break;
 
     case IP_DEST_HOST_UNREACHABLE:
-        OutputText(IDS_TIMEOUT);
-        break;
+    case IP_DEST_NET_UNREACHABLE:
+        FoundTarget = true;
```

Comment:
I assumed this was incorrect, but windows does indeed return success (0) if it fails to find the target. Weird.